OM-Demo/ImageViewController.swift

```diff
@@ -45,26 +45,24 @@ class ImageViewController: BaseAdUnitViewController {
 
         //Create verification resource using the values provided in the ad response
         guard let verificationResource = createVerificationScriptResource(vendorKey: vendorKey,
-                                                                          verificationScriptURL: urlToMeasurementScript,
+                                                                          verificationScriptURL: urlToMeasurementScript.absoluteString,
                                                                           parameters: parameters)
             else {
                 fatalError("Unable to instantiate session context: verification resource cannot be nil")
         }
 
         //Create native ad session context
         do {
-            return try OMIDDemobuildAdSessionContext(partner: partner, script: omidJSService, resources: [verificationResource], customReferenceIdentifier: nil)
+            return try OMIDAdSessionContext(partner: partner, script: omidJSService, resources: [verificationResource], contentUrl: nil, customReferenceIdentifier: nil)
         } catch {
             fatalError("Unable to instantiate session context: \(error)")
         }
     }
 
-    override func createAdSessionConfiguration() -> OMIDDemobuildAdSessionConfiguration {
+    override func createAdSessionConfiguration() -> OMIDAdSessionConfiguration {
         //Create ad session configuration
         do {
-            return try OMIDDemobuildAdSessionConfiguration(impressionOwner: .nativeOwner,
-                                                         videoEventsOwner: .noneOwner,
-                                                         isolateVerificationScripts: false)
+            return try OMIDAdSessionConfiguration(creativeType: .nativeDisplay, impressionType: .viewable, impressionOwner: .nativeOwner, mediaEventsOwner: .noneOwner, isolateVerificationScripts: true)
```

Comment:
Can you also include the display loaded event if the impressionOwner is native? 
Should come before the impression.

Reply:
done.